operator/internal/manifests/openshift/opa_openshift.go

```diff
@@ -38,14 +38,17 @@ func newOPAOpenShiftContainer(mode lokiv1.ModeType, secretVolumeName, tlsDir, mi
 	uriScheme = corev1.URISchemeHTTP
 	args = []string{
 		"--log.level=warn",
-		"--opa.skip-tenants=audit,infrastructure",
-		"--opa.admin-groups=system:cluster-admins,cluster-admin,dedicated-admin",
 		fmt.Sprintf("--web.listen=:%d", GatewayOPAHTTPPort),
 		fmt.Sprintf("--web.internal.listen=:%d", GatewayOPAInternalPort),
 		fmt.Sprintf("--web.healthchecks.url=http://localhost:%d", GatewayOPAHTTPPort),
+		"--opa.skip-tenants=audit,infrastructure",
 		fmt.Sprintf("--opa.package=%s", opaDefaultPackage),
 	}
 
+	if len(adminGroups) > 0 {
+		args = append(args, fmt.Sprintf("--opa.admin-groups=%s", adminGroups))
+	}
```

Comment:
This is the solution when a user unsets the admin groups entirely, right?

Reply:
Yes, but it's only a part of the logic.